themes/codefor-theme/layouts/partials/nav.html

```diff
@@ -17,13 +17,13 @@
       <ul class="navbar-nav justify-content-around mr-3 mt-4 mt-lg-0">
         {{ $currentPage := . }}
         {{ range .Site.Menus.main }}
-        {{if ne .Weight 0}}
-        <li class="{{ if or ($currentPage.IsMenuCurrent "main" .) ($currentPage.HasMenuCurrent "main" .) }}active{{ end }}">
-            <a class="nav-link px-3" href="{{ .URL }}" title="{{ .Title }}">{{ .Name }}</a>
-        </li>
-        {{ end }}
+          {{if ne .Weight 0 }}
+          <li class="{{ if or ($currentPage.IsMenuCurrent "main" .) ($currentPage.HasMenuCurrent "main" .) }}active{{ end }}">
+              <a class="nav-link px-3" href="{{ .URL }}" title="{{ .Title }}">{{ .Name }}</a>
+          </li>
+          {{ end }}
         {{ end }}
       </ul>
     </div>
   </nav>
-</div>
+</div>
```

Comment:
@iamsilvio kannst du deinen Editor so einstellen dass es diese Art von Änderungen nicht mit die Commits schafft?

Reply:
Mein Formatter hat das newline in der Zeile rausgeschmissen, hab keine codeing Guidelines fürs Projekt an die ich mich halten könnte, aber wenn das für euch wichtig ist, mach ich ne Projekt spezifische config!

Reply:
Das wäre fein